lib/Doctrine/ORM/AbstractQuery.php

```diff
@@ -401,8 +401,8 @@ public function setParameter($key, $value, $type = null)
      *
      * @param mixed $value
      *
-     * @return mixed[]|string|int|float|bool
-     * @psalm-return array|scalar
+     * @return mixed[]|string|int|float|bool|object|null
+     * @psalm-return array|scalar|object|null
```

Comment:
I've seen `DateTime`, `CustomIdObject` and `null` pass here when running unit tests with a native type declaration.

Reply:
That's basically `mixed`, isn't it?

Reply:
Oh crap didn't see your answer… well I guess you can't have `resource` in there but yeah, mixed might be better here :sweat_smile: